com.lixgame.Lix.json

```
@@ -0,0 +1,105 @@
+{
+    "app-id": "com.lixgame.Lix",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
+    ],
+    "command": "lix",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
```

```suggestion
        "--socket=fallback-x11",
        "--socket=wayland",
```

assuming it supports wayland

---

```
object.Exception@src/hardware/display.d(89): Can't instantiate a display even though we want to run Lix in interactive mode. This is very strange; normally, at least a window of size 640x480 should have spawned.
----------------
??:? pure @safe noreturn std.exception.bailOut!(Exception).bailOut(immutable(char)[], ulong, scope const(char)[]) [0x55d5453c3e1a]
??:? pure @safe allegro5.display.ALLEGRO_DISPLAY* std.exception.enforce!().enforce!(allegro5.display.ALLEGRO_DISPLAY*).enforce(allegro5.display.ALLEGRO_DISPLAY*, lazy const(char)[], immutable(char)[], ulong) [0x55d5453de7b4]
??:? void hardware.display.setScreenMode(in basics.cmdargs.Cmdargs) [0x55d54546b1da]
??:? void basics.resol.changeResolutionBasedOnCmdargsThenUserFile(const(basics.cmdargs.Cmdargs)) [0x55d54539d5bd]
??:? void basics.init.initialize(in basics.cmdargs.Cmdargs) [0x55d54539c5f6]
??:? int main.main(immutable(char)[][]).__lambda3() [0x55d54548cb40]
??:? extern (C) int allegro5.system.al_run_allegro(scope int delegate()).main_runner(int, char**) [0x55d5454d6f9d]
??:? int allegro5.system.al_run_allegro(scope int delegate()) [0x55d5454d6f7d]
??:? _Dmain [0x55d54548cb10]
```

Looks like Allegro 5 is not compatible with Wayland.